src/renderer/components/item-object-list/item-list-layout-header/item-list-layout-header-info/item-list-layout-header-info.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import React from "react";
+import { observer } from "mobx-react";
+import type { ItemObject, ItemStore } from "../../../../../common/item.store";
+import type { Filter } from "../../page-filters.store";
+import type { HeaderPlaceholders } from "../../item-list-layout";
+
+interface ItemListLayoutHeaderInfoProps<I extends ItemObject> {
+  headerPlaceholders: HeaderPlaceholders;
+  getItems: () => I[];
+  store: ItemStore<I>;
+  getFilters: () => Filter[]
+  toggleFilters: () => void
+}
+
+export const ItemListLayoutHeaderInfo = observer(
+  <I extends ItemObject>({
+    headerPlaceholders,
+    getItems,
+    getFilters,
+    store,
+    toggleFilters,
+  }: ItemListLayoutHeaderInfoProps<I>) => {
+    const renderInfo = () => {
+      const allItemsCount = store.getTotalCount();
+      const itemsCount = getItems().length;
```

Comment:
_nit:_ I'd better to move destructing input params inside constructor (more readable):

```suggestion
export const ItemListLayoutHeaderInfo = observer(
  <I extends ItemObject>(props: ItemListLayoutHeaderInfoProps<I>) => {
    const { headerPlaceholders, getItems, getFilters, store, toggleFilters, } = props;
```

Reply:
Updated